src/main/java/org/bookstore/dto/request/CreateBookRequestDto.java

```diff
@@ -22,8 +25,12 @@ public record CreateBookRequestDto(
         @Positive
         BigDecimal price,
 
+        @Size(max = 1000)
         String description,
 
-        String coverImage
+        @Size(max = 255)
+        String coverImage,
+
+        List<Long> categoryIds
```

Comment:
```suggestion
        @NotEmpty
        List<Long> categoryIds
```
Can a book exist without any category?